src/server/main_service.cc

```diff
@@ -1204,8 +1202,7 @@ class ReplyGuard {
   SinkReplyBuilder* builder_ = nullptr;
 };
 
-bool Service::InvokeCmd(const CommandId* cid, CmdArgList tail_args, ConnectionContext* cntx,
-                        bool record_stats) {
```

Comment:
removed `record_stats` and made move the logic into `InvokeCmd`. Also simplified the logic imho.